man/SAPHanaSR_maintenance_examples.7

```diff
@@ -158,9 +183,9 @@ If everything looks fine, proceed.
 .RE
 .RS 4
 .br
-~> cdpy; python ./systemReplicationStatus.py; echo RC:$?
+~> HDBsettings.sh systemReplicationStatus.py; echo RC:$?
 .br
-~> cdpy; python ./landscapeConfigurationStatus.py; echo RC:$?
+~> HDBsettings.sh ./landscapeConfigurationStatus.py; echo RC:$?
```

Comment:
Not sure if this will work. Better to remove the `./` in front of landscapeConfigurationStatus.py if we go back to the syntax with `HDBsettings.sh` instead of `cdpy; python`.